nautobot/dcim/factory.py

```diff
@@ -698,6 +698,7 @@ class Params:
     default_image = factory.LazyAttribute(
         lambda o: not o.software_version.software_image_files.filter(default_image=True).exists()
     )
+    external_integration = random_instance(ExternalIntegration, allow_null=False)
```

Comment:
Since it's a nullable ForeignKey, I'd expect `allow_null=True` here to provide appropriate diversity of data?